### PR TITLE
Support add users and groups to an ipa idm

### LINKED ISSUE #62: Add ability to create identities in an IDM
Must be able to add users to an idm
Must be able to add groups to an idm
Must be able to add users to groups in an idm

### PR DESCRIPTION
**Pull request by mcanoy**
#### What does this PR do?
Exact same code as PR 62 -- previously reverted. Provides the ability to create users and groups in an idm (RHEL IPA) in a new role ```create-identities```

#### Which tests illustrate how this code works?
roles/create-identities/tests/create_idm.yml

#### Is there a relevant Issue open for this?
Resolves #62 

#### Are there any other relevant resources that should be reviewed as well?
n/a

#### Who would you like to review this?
/cc @oybed  @sherl0cks

### CODE REVIEW

File: roles/create-identities/tests/inventory [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[ipa]
+mcanoy-idm.core.rht-labs.com ansible_user=admin
```
Comment (oybed) [inline roles/create-identities/tests/inventory]: please remove this file or change the FQDN to something less lab specific

File: roles/create-identities/tests/create_idm.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (oybed) [inline roles/create-identities/tests/create_idm.yml:1]: Modeled after this, could we create a more general playbook at the top level playbooks directory (see `load_infra.yml` for an example) and add it to the README? I would like to see us generate a set of playbooks that a fit for purpose and easy to adopt/use with other management tooling, such as Tower.

**Review by oybed — COMMENTED** [review]
@mcanoy looks good for the most part. A couple of comments. We can jump on a call to discuss more details if you'd like to.

File: roles/create-identities/tests/inventory (hunk 1 above)
Comment (mcanoy) [reply roles/create-identities/tests/inventory]: ok. modified to my-idm.com no user specified.

File: roles/create-identities/tests/create_idm.yml (hunk 2 above)
Comment (mcanoy) [reply roles/create-identities/tests/create_idm.yml:1]: yep. updated this

**Comment by oybed** [comment]
LGTM - ship it!

**Comment by oybed** [comment]
@sherl0cks go ahead and merge if you are ok with this still. Note the additional changes - including the playbook at the top level directory.

**Pull request merged**